source/common/statistic_impl.cc

```diff
@@ -236,6 +246,31 @@ nighthawk::client::Statistic HdrStatistic::toProto(SerializationDomain domain) c
   return proto;
 }
 
+absl::StatusOr<std::unique_ptr<std::istream>> HdrStatistic::serializeNative() const {
+  char* data;
+  if (hdr_log_encode(histogram_, &data) == 0) {
+    absl::string_view s(data, strlen(data));
+    auto ss = std::make_unique<std::stringstream>();
```

Comment:
Can we use more descriptive names instead of just `s` and `ss`?

Reply:
Done in 79bad46